src/connector.js

```diff
@@ -345,13 +346,7 @@ module.exports = class Connector extends events.EventEmitter {
   _initialise() {
     this.query( this.statements.initDb( this.options.schema ), ( error, result ) => {
       if( error ) {
-        // retry for errors caused by concurrent initialisation
-        if( error.code === INTERNAL_ERROR ) {
-          this._initialise()
-          return
-        } else {
-          throw error
-        }
+        return this._initialise()
```

Comment:
Let me add the checks back so it only tries to reconnect on `INTERNAL_ERROR`, `DATABASE_IS_STARTING_UP` and `CONNECTION_REFUSED`.